services/tvl/docker-compose.yml

```diff
@@ -13,25 +13,31 @@ networks:
 services:
   api:
     build: .
+    image: yearn-exporter
     entrypoint: uvicorn yearn.api:app --host 0.0.0.0 --port 4000
     ports: 
       - 4000:4000
     environment:
-      - PGUSER=postgres
       - PGHOST=postgres
+      - PGDATABASE=yearn
+      - PGUSER=postgres
+      - PGPASSWORD=yearn
     restart: always
     networks:
       - yearn-exporter-tvl
   yearn-exporter-tvl:
     build: .
+    image: yearn-exporter
     command: historical_tvl
     environment:
       - WEB3_PROVIDER
       - ETHERSCAN_TOKEN
       - EXPLORER
-      - PGUSER=postgres
       - PGHOST=postgres
-      - PGPASS=yearn
+      - PGDATABASE=yearn
+      - PGUSER=postgres
+      - PGPASSWORD=yearn
+      - ENV
```

Comment:
not sure `ENV` is needed anymore, it has been deprecated in favor of `SENTRY_ENVIRONMENT`, pls grep for it and remove all occurences.

Reply:
done. there was only one occurance